src/platform/platform_posix.c

```diff
@@ -514,7 +516,11 @@ CxPlatGetAbsoluteTime(
 
     CxPlatZeroMemory(Time, sizeof(struct timespec));
 
+#if defined(CX_PLATFORM_LINUX)
     ErrorCode = clock_gettime(CLOCK_MONOTONIC, Time);
+#elif defined(CX_PLATFORM_DARWIN)
```

Comment:
I'm not sure why you need this. clock_gettime() should work on macOS just fine AFAIK.

Reply:
If I recall correctly, the issue I was experiencing was that clock_gettime was actually *not* monotonic (Seems I wasn't the only one: https://github.com/mobile-shell/mosh/pull/1116), which was breaking some of the pthread event waiters. I feel like my initial PR used mach_absolute_time...

Reply:
I would add note there in that case as it is not obvious. .NET runtime uses clock_gettime AFAIK and I'm not aware of any issue. It may not use assert or use that output in the same way.